Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md

```diff
@@ -1,2 +1,26 @@
 To allow us access to Microsoft Graph User, an admin has to approve our app using an admin consent flow, by clicking on the following [link](https://oproxy.demisto.ninja/ms-graph-user).
-After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
\ No newline at end of file
+After authorizing the Demisto app, you will get an ID, Token, and Key, which should be inserted in the integration instance configuration's corresponding fields.
+## Authorize Cortex XSOAR for Microsoft Graph User
```

Comment:
it is not enough to add it only in the detailed description, need to add it also to the documentation
lets add it to https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication which is already referred from the integration docs

Reply:
There is a part for a self deployed app, and each integration has its own required permissions (scope) so I'm not really sure what you are after..
Is it just re writing this section in a more detailed way?